### ISSUE TITLE
Decoupling - Without loosing log bootstraping

### ISSUE THREAD

**Issue by csrui** [log-oscon/WP-Docker-Construct#4]
After looking at all the stuff available here, I have an issue about some assumptions made out of the box. Yes, I get that this is a bootstrap for log projects, but not all log projects are build the same way :)

Addressing some points would allow for more flexibility while still retaining all the current features.

Taking into account that this is a fork here's what we can do better.

## Step 1 - Decouple WordPress "website" from Docker Setup.

Since we are dealing with the age of programmable infrastructure I see the contents of this repository as two separate projects. 1 - The Docker and 2 - The WordPress.
By separating the two as much as possible, updates from 10up will be easier as well as updates on every cloned project out there (less conflicts).

## Step 2 - setup.sh should be as dumb as possible
**bin/setup.sh** makes some assumptions on how the build process is supposed to go. I believe that this file should be as dumb as possible.

It's easily attainable by extracting the logic into a project specific build file. PR https://github.com/log-oscon/WP-Docker-Construct/pull/2

## Step 3 - Getting the WordPress project into the wordpress folder.

I purpose that for example a bin/setup.sh (or a separate file.sh) does a git clone of the actual WordPress boilerplate contents exemplified on PR https://github.com/log-oscon/WP-Docker-Construct/pull/3

**Comment by Narayon** [comment]
I agree with this since it decouples the structural system from the starter(WP Construct).  

Before we can go forward, we need to change the WP-Construct to work with this changes and create a new repo for the VVV structure.  

I think we only need to remove the structure configuration files from the WP install folder, the same way is already done in this repo.

Great suggestions @csrui

**Comment by Narayon** [comment]
@csrui I'm starting to clean the WP-Construct repo.  
Can you do the same in this repo?  
We just need to remove the preinstalled WP-Construct and update the README.

**Comment by csrui** [comment]
Sure.

Will try to do it in a bit.

**Comment by csrui** [comment]
Checkout PR https://github.com/log-oscon/WP-Docker-Construct/pull/5

**Comment by Narayon** [comment]
Great job, @csrui.  
It's just missing the documentation of the install process (what files to edit and with what options).  
I think we should remove elasticsearch or, at least, make it an option but not default.

**Comment by csrui** [comment]
I'm still a bit unsure about what to do on the documentation but I'll close this issue.
